Sources/Gui/Main.cpp

```diff
@@ -57,13 +57,23 @@
 #endif
 
 
+#ifdef _MSC_VER>=1900	// Visual Studio 2015 or higher
+extern "C" { FILE __iob_func[3] = { *stdin,*stdout,*stderr }; }
```

Comment:
Thank you for taking your time for this pull request.

> Edit: I suppose I should remove the changes to the images' loading now, seeing as it works that way, right?
Yeah, please revert those changes and push again.